test/common/odp_cunit_common.c

```diff
@@ -707,9 +711,18 @@ int odp_cunit_register(odp_suiteinfo_t testsuites[])
 int odp_cunit_parse_options(int argc, char *argv[])
 {
 	const char *env = getenv("CI");
+	int i;
 
 	progname = argv[0];
-	odph_parse_options(argc, argv);
+	argc = odph_parse_options(argc, argv);
```

Comment:
It could be simpler to integrate this to CI by using an environment variable. There is already the `CI` variable, so for example `CI_CONTROL_THREAD`/`CI_THREAD_TYPE_CONTROL`/`CI_DEFAULT_THREAD_TYPE_CONTROL` could be used.

Reply:
Ok.. Can I change the default type to WORKER which can be overridden using this ENV variable ?

Reply:
And there are lot of tests test/validation/api/scheduler/, timer, pktio
which do lot of multi thread perf tests.

So I think we need to change the default with worker thread.

Reply:
Yes, changing the default type to worker is OK. We can then add a separate test case to CI, which changes the type to CONTROL using an env variable.